packages/flutter_secure_storage/example/integration_test/flutter_secure_storage_test.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+String _randomValue() {
+  final rand = Random();
+  final codeUnits = List.generate(20, (index) {
+    return rand.nextInt(26) + 65;
+  });
+
+  return String.fromCharCodes(codeUnits);
+}
+
+Future<void> main() async {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+  late FlutterSecureStorage storage;
+  late final Map<String, String> testInput = {};
+
+  setUpAll(() async {
+    storage = const FlutterSecureStorage();
+
+    for (int i = 0; i < 5; i++) {
+      testInput[_randomValue()] = _randomValue();
```

Comment:
Do not use random strings as test input. Tests should always generate the same result.